starport/services/scaffolder/type.go

```diff
@@ -101,15 +151,15 @@ func (s *Scaffolder) AddType(
 
 	// create the type generator depending on the model
 	// TODO: rename the template packages to make it consistent with the type new naming
```

Comment:
Just to know, do you plan to make this change in this PR? Because I was thinking about doing it

Reply:
No really, I want to keep this PR small. But definitely, templates pkg needs an overall refactoring.